app/src/ui/organism/header/HeaderBottomSection.js

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { ReactComponent as ArrowDownIcon } from "../../../assets/img/arrow-down.svg";
+import { ReactComponent as ToogleMenuIcon } from "../../../assets/img/toogle-menu.svg";
+import { ReactComponent as ArrowRightIcon } from "../../../assets/img/arrow-right.svg";
+
+const HeaderBottomSection = () => {
+  //rozwijanie listy
+  const [visible, setVisible] = useState(false);
+  const popUp = visible ? "" : "visually-hidden";
+  //pobieranie danych z servera
+  const [categories, setCategories] = useState([]);
```

Comment:
all state variables and functions to set those variables should be on top of the component so 
const [visible, setVisible] = useState(false);
const [categories, setCategories] = useState([]);
then we know where to find all function/variables responsible for state managment